### ISSUE TITLE
Allow exporting animations as 1 row and a lot of columns

### ISSUE THREAD

**Issue by mak448a** [viniciusgerevini/godot-aseprite-wizard#175]
Could you add exporting the sprites in a single row?

The grid is messing up the animations because it shows a bar on top of the animations. When it used to be a bunch of sprites in a line in my project, it worked fine.

The bug in godot engine: https://github.com/godotengine/godot/issues/76435

**Comment by viniciusgerevini** [comment]
Hello. What is the method you are using to import your animations? Is it via Inspector doc?

**Comment by mak448a** [comment]
Yes.

**Comment by viniciusgerevini** [comment]
I'll work on exposing this configuration. One thing to keep in mind though is that exporting the spritesheet in a single row does not solve the issue completely and depending on how many frames you have you might reach the image width limit (16384px). This is very unlikely if you use a small canvases, but still a possibility.

Usually people workaround this issue by adding a padding of a few pixels to the file. That might be the quickest workaround without much downside.

**Comment by mak448a** [comment]
Is there a way to add the padding in godot-aseprite-wizard?

**Comment by viniciusgerevini** [comment]
No. That's best done in the Aseprite file itself. I might check if the CLI options work with the current implementation, but as of today that's not a feature.

**Comment by cwchunwei** [comment]
I would like to have this feature as well. My use case is when I have an animation of let's say 3 frames. The wizard would pack this in a 2x2 format, and there would be wasted space in the bottom right corner. It's not the biggest problem, but for optimization sake, it would be nice to not have this empty space.
